clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/model/ResizeAzureServerGroupDescription.groovy

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2014 Netflix, Inc.
```

Comment:
s/2014/2019
s/Netflix, Inc./The original authors

Reply:
Thanks. Updated to "Copyright 2019 The original authors."